eslint.config.js

```diff
@@ -1,3 +1,9 @@
 import { solvro } from "@solvro/config/eslint";
 
-export default solvro();
+export default solvro({
+  ignores: [
+    "src/components/ui/*.ts?(x)",
+    "src/components/ui/minimal-tiptap/*.ts?(x)",
+    "src/components/ui/minimal-tiptap/**/*.ts?(x)",
+  ],
```

Comment:
to można uprościć wykluczając cały folder `ui`, co usunie potrzebę dodawania nowych folderów jeśli w przyszłości dodamy jakieś inne rozbudowane komponenty do podfolderów

```suggestion
  ignores: ["src/components/ui/*.ts?(x)", "src/components/ui/**/*.ts?(x)"],
```